### PR TITLE
Rename-assertions

### PR DESCRIPTION
**Pull request by alencar13**
It was excluded one assertion because was duplicated and renamed one assertion.

### CODE REVIEW

File: sources/index.spec.js [hunk 1]
```diff
@@ -184,8 +180,8 @@ describe('chai-style', () => {
     })
   })
 
-  describe('Different order of values', () => {
-    it('test box-shadow', () => {
+  describe('Differents order of pixels values', () => {
+    it('box-shadow test', () => {
```
Comment (darlanmendonca) [inline sources/index.spec.js]: maybe this assertion be better inside of describe "values", line 59

Comment (darlanmendonca) [reply sources/index.spec.js]: just a it function, with string like "should assert with any order of value"

Comment (darlanmendonca) [reply sources/index.spec.js]: how in line 25, the css applied is "0 0 10px red", in this spec, maybe be better write something very different in order, like "#f00 0 0 10px" , to be explicit, when anyone read this spec (legibility)

**Review by darlanmendonca — CHANGES_REQUESTED** [review]

**Comment by alencar13** [comment]
We already have an assertion (properties) that use camelCase and works well but in the element `boxshadow` but if we use this same element with camelCase is crashing as error below:

``` js
 ✖ should assert with any order of value 
         Electron 1.6.6 (Linux 0.0.0)
       AssertionError: expect div to have {boxShadow: #F00 0px 0px 10px 0px}, is receiving {boxShadow: rgb(255, 0, 0) 0px 0px 10px 0px}
           at Context.it (/tmp/sources/index.spec.js:95:0 <- /tmp/04f59fc04a48ba6526ff2c039b277c88.browserify:22734:31)
```

Please, check it.

**Comment by darlanmendonca** [comment]
checking now

**Review by darlanmendonca — APPROVED** [review]

**Comment by darlanmendonca** [comment]
approved, 🚀

**Pull request merged**